### ISSUE TITLE
Insertion of a node js Buffer Object in a binary field

### ISSUE THREAD

**Issue by khalilTN** [balderdashy/waterline#813]
I'm trying to insert a buffer object (node js buffer) into a binary field, (BLOB in mysql), i found that the buffer is arriving to Mysql Adapter in another structure and the data is corrupted.

**Comment by dmarcelino** [comment]
Hi @khalilTN, can you please give code and model definition examples?

sails-mysql passes the [binary test](https://github.com/balderdashy/waterline-adapter-tests/blob/master/interfaces/semantic/types/binary.type.js), excerpt below:
``` javascript
        // use a string
        var str = 'test the things';
        // to make a binary thing
        var buf = new Buffer(str, "utf-8");
        // store the binary thing
        Semantic.User.create({ avatar: buf }, function(err, record) {
          assert(!err, err);
          // read out the stored binary thing
          var outbuf = new Buffer(record.avatar);
          
          assert(outbuf.toString('utf-8') === str);
          done();
        });
```

**Comment by devinivy** [comment]
@khalilTN does this error persist for you?

**Comment by khalilTN** [comment]
no it's is corrected, it was sails-MySQL issue, and was corrected in the 0.10.8 version

**Comment by devinivy** [comment]
Glad!  :)
